docs/pages/bare/installing-expo-modules.md

```diff
@@ -10,15 +10,15 @@ import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
 
 In order to use Expo modules in your app, you will need to install and configure the `expo` package.
 
-The `expo` package has a small footprint; it includes only a minimal set of packages that are needed in nearly every app and the module and autolinking infrastructure that other Expo SDK packages are built with. Once the `expo` package is installed and configured in your project, you can use `expo install` to add any other Expo module from the SDK.
+The `expo` package has a small footprint; it includes only a minimal set of packages that are needed in nearly every app and the module and autolinking infrastructure that other Expo SDK packages are built with. Once the `expo` package is installed and configured in your project, you can use `npx expo install` to add any other Expo module from the SDK.
 
 The easiest way to get started with Expo modules is to initialize a new bare workflow project with: `npx create-expo-app --template bare-minimum`.
 
 If you have an existing project without Expo modules installed (perhaps created with `npx react-native init`), please follow the automatic or manual installation instructions below.
```

Comment:
I think we should re-write this. This sounds like a duplication of info that is already covered in "Overview".

It's better to say
```suggestion
Depending on how you [initialized the project](/bare/hello-world/), there are two ways you can install the Expo modules: automatic or manual.
```